dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -83,6 +84,58 @@ public abstract class ClientBase
         return response.Value.Choices[0].Text;
     }
 
+    protected async IAsyncEnumerable<string> InternalCompletionStreamAsync(
```

Comment:
I'm not clear on the need the "Internal" version instead of just implementing this in CompleteStreamAsync(). I see you're following an existing pattern in this class. Is this needed?

Reply:
Actually the internal version is a wrap because the same implementation exists for both Azure Open AI and Open AI and Azure provides a client that is also compatible with Open AI directly.

I just updated the PR using the Internal also in the OpenAI and it worked as expected.